src/whats-new.md

```diff
@@ -178,6 +176,10 @@ to reflect the latest available codelabs.
 We've published the following articles on the [Flutter Medium][]
 publication since the last stable release:
 
+* [What's next for Flutter][]
```

Comment:
I don't see the URL for this below. Was it already in the content?

Reply:
```console
Checking for valid link references...
check_link_references: Invalid link references found!

whats-new/index.html:
[What’s new in Flutter 3.7][]
make[1]: *** [Makefile:120: build-image-and-check-links] Error 1
make[1]: Leaving directory '/home/runner/work/website/website'
make: *** [Makefile:136: build] Error 2
```

Reply:
yup!